camel/toolkits/google_maps_toolkit.py

```diff
@@ -157,8 +179,8 @@ def get_address_description(
             ImportError: If the `googlemaps` library is not installed.
             Exception: For unexpected errors during the address validation.
         """
-        googlemaps = self._import_googlemaps_or_raise()
-        google_maps_api_key = self._get_googlemaps_api_key()
+        googlemaps = _import_googlemaps_or_raise()
+        google_maps_api_key = _get_googlemaps_api_key()
         try:
             gmaps = googlemaps.Client(key=google_maps_api_key)
         except Exception as e:
```

Comment:
can we move this Client setting init into `__init__`?, for the checking we can use `@dependencies_required` and `@api_keys_required`